test/specs/commonTests.js

```diff
@@ -486,27 +486,26 @@ export const rendersChildren = (Component, options = {}) => {
 // ----------------------------------------
 // className from prop
 // ----------------------------------------
-const _noDefaultClassNameFromProp = (Component, propKey, options = {}) => {
+const _noDefaultClassNameFromProp = (Component, propKey, propValues, options = {}) => {
+  const { defaultProps = {} } = Component
   const { className = propKey, requiredProps = {} } = options
   // required props may include a prop that creates a className
   // if so, we cannot assert that it doesn't exist by default because it is required to exist
   // skip assertions for required props
+  if (propKey in defaultProps) return
```

Comment:
I've rebased, but tests failed on `Sidebar`, it has `defaultProps` for `direction`. So, I think it's expected behaviour there.